File: sdkwork-ai-sdk-go/types/suno_music.go
package types

// Suno音乐对象
type SunoMusic struct {
	Id string `json:"id"`
	Object string `json:"object"`
	Title string `json:"title"`
	Status string `json:"status"`
	Duration int `json:"duration"`
	Tags []string `json:"tags"`
	Genre string `json:"genre"`
	AudioUrl string `json:"audioUrl"`
	Lyrics string `json:"lyrics"`
	Instrumental bool `json:"instrumental"`
	Error MusicError `json:"error"`
	CreatedAt int `json:"created_at"`
	CompletedAt int `json:"completed_at"`
}

File: sdkwork-ai-sdk-csharp/Models/SunoMusic.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Ai.Models
{
    public class SunoMusic
    {
        public string? Id { get; set; }
        public string? Object { get; set; }
        public string? Title { get; set; }
        public string? Status { get; set; }
        public int? Duration { get; set; }
        public List<string>? Tags { get; set; }
        public string? Genre { get; set; }
        public string? AudioUrl { get; set; }
        public string? Lyrics { get; set; }
        public bool? Instrumental { get; set; }
        public MusicError? Error { get; set; }
        public int? CreatedAt { get; set; }
        public int? CompletedAt { get; set; }
    }
}
